ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -109,12 +109,12 @@ ${listenerText}
    */
   get installBoundaryText() {
     const configPath = this.configFilePath || '<path>';
-    const ossContent = `curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add - ;\\
+    const ossContent = `curl -sS https://api.releases.hashicorp.com/v1/releases/boundary/latest | jq '.version';\\
```

Comment:
@randallmorey do we need the json processor command?

Reply:
I think we want it, so we alway return the latest version in JSON.

Reply:
Is it necessary? It would require the user to have `jq` and error out if they don't have it

Reply:
@ZedLi that was my thought as well, wouldn't it say command not found?

Reply:
Yeah, I personally don't think we need the user to have `jq` just to output the version, is there that much output that necessitates it?

Reply:
I misunderstood what the purpose of this endpoint was, we need jq to parse out the version to grab the correct download